charts/rhtap-dh/templates/app-config-content.yaml

```diff
@@ -54,6 +54,12 @@ catalog:
       - Location
       - Template
       - API
+jenkins:
```

Comment:
The new plugins need to be gated based on the presence of the `rhtap-jenkins-integration`  secret. See how it's done for github.